Prac05/count_words.py

```diff
@@ -0,0 +1,32 @@
+"""
+Program asks user for a string containing several words, then counts how many times each word appears in the string.
+Text: this is a collection of words of nice words this is a fun thing it is
+"""
+
+user_string = input("Enter the words for your string: ")
+string_list = user_string.split()
+
+count_word = {}
+
+for word in string_list:
+    if word in count_word:
+        count_word[word] += 1
+    else:
+        count_word[word] = 1
+
```

Comment:
When you are doing the loop above you could also find the longest word in one go so you don't have to loop over it again below